src/flag_gems/ops/cross_entropy_loss.py

```diff
@@ -121,7 +120,7 @@ def celoss_probability_kernel(
         w_ptrs = w_ptr + offset_c
         w_mask = offset_c < C
         inp = tl.load(inp_ptrs, mask, other=0).to(tl.float32)
-        tgt = tl.load(tgt_ptrs, mask, other=1).to(tl.float32)
+        tgt = tl.load(tgt_ptrs, mask, other=0).to(tl.float32)
```

Comment:
Since `w` is also masked on `C` dimension, and tgt only affects the result by pointwise-multiplying w, this change has no effect on outputs?

Reply:
actually there is no effect. setting the default value as 0 is more elegant in my view ;)